src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs

```diff
@@ -132,19 +141,15 @@ private async Task CreateBlocksAsync()
                         continue;
                     }
 
-                    uint timeNow = (uint) this.dateTimeProvider.GetAdjustedTimeAsUnixTimestamp();
-
-                    if (timeNow <= this.consensusManager.Tip.Header.Time)
+                    if ((uint)this.dateTimeProvider.GetAdjustedTimeAsUnixTimestamp() <= this.consensusManager.Tip.Header.Time)
                     {
                         await this.TaskDelayAsync(500).ConfigureAwait(false);
```

Comment:
Use `TimeSpan.FromMillisecond(500)` instead of `500`, it is more explicit

Reply:
Not sure if needed. Everyone is used to methods of such functionality taking arguments in milliseconds (Task.Delay, Thread.Sleep). So I just think using `TimeSpan.FromMillisecond(500)` takes more space and is unnessesary

Reply:
well I actually think its more readable to use `FromMillisecond` this anyways gets optimised by the compiler, but its not big deal just using 500 is clear its ms.

Reply:
Ok, fixed